src/MainMenu.cpp

```diff
@@ -2363,5 +2363,50 @@ ZunResult MainMenu::LoadDiffCharSelect(MainMenu *menu)
 }
 #pragma optimize("", on)
 
+#pragma optimize("s", on)
+void th06::MainMenu::SelectRelated(MainMenu *menu, i16 btn_pressed, i16 old_mapping, ZunBool param_4)
```

Comment:
We use camelCase everywhere, so it'd be nice to rename those to btnPressed and oldMapping.